start-client/src/styles/_main.scss

```diff
@@ -196,6 +196,9 @@ body {
       border: 0;
       padding: 4px;
       outline: none;
+      &:hover {
```

Comment:
The button in the non-mobile view has `cursor: pointer;` without the hover pseudoclass on it. Can we remove the pseudoclass here or did I overlook something?

Reply:
Yes, we can directly use this cursor: pointer; under outline: none;
It will do the same job.

Reply:
Great, then please let's do that for consistency.